### PR TITLE
Showcase proofreading

### PR DESCRIPTION
**Pull request by atravitz**
Minor changes to address things that I noticed while running the showcase notebook. 

I also cleared the notebook output to make the diffs easier to track in the future. Please let me know if this is not best-practice.

### CODE REVIEW

File: showcase/openfe_showcase.ipynb [hunk 1]
```diff
@@ -478,140 +436,15 @@
...
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "# Visualize the mapping in 3D\n",
     "from openfe.utils import visualization_3D\n",
     "\n",
-    "visualization_3D.view_mapping_3d(lomap_mapping)"
+    "visualization_3D.view_mapping_3d(lomap_mapping, show_atomIDs=True))"
```
Comment (mikemhenry) [inline showcase/openfe_showcase.ipynb:447]:
```suggestion
    "visualization_3D.view_mapping_3d(lomap_mapping, show_atomIDs=True)"
```
Looks like an extra `)` got added

**Comment by mikemhenry** [comment]
I do think that it is best practice to not keep the output BUT we need the output so that our docs render the output:
https://docs.openfree.energy/en/stable/tutorials/rbfe_python_tutorial.html

I think what we should do is maybe have CI do the rendering (it already does the running (at least on these notebooks https://github.com/OpenFreeEnergy/ExampleNotebooks/blob/main/.github/workflows/CI.yml#L68 we will want to make sure those are a 1-1 match of the notebooks we have in our docs))
We could have the result of running the notebooks committed on merge into main? That way it won't clutter up the PR (with each CI run you would need to pull changes) BUT I don't think that would actually help since the output would still be saved... 

We could have RTD keep the output which would fix the issue of the docs not having the output BUT that means that if you clone the repo or view the repo with a web browser there still would be no output.

I am not really sure what we should do here, maybe see what OpenFF does? They have the same-ish setup for embedding notebooks in their docs.

This will probably be a bigger issue to work out BUT I think the typo fixes you have here are great :tada: so could you include the output for now? I will turn this comment into an issue that we can use to discuss what we want to do RE: notebook output being committed.

**Review by IAlibay — CHANGES_REQUESTED** [review]
I'm going to block over outputs - needs further discussion.

**Comment by IAlibay** [comment]
<p>In this case it's fine, although you are right - this is a long standing issue that our network generations aren't deterministic.</p>

---

 <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/163/discussion/#comment-2403490477'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by IAlibay — APPROVED** [review]
I'll approve, but please do look at my reviewnb comments.

**Comment by atravitz** [comment]
<p>I think you're right, I was thinking of it in the "uncountable" way. <a href="https://dictionary.cambridge.org/grammar/british-grammar/content-or-contents" target="_blank">https://dictionary.cambridge.org/grammar/british-grammar/content-or-contents</a>. Fixed.</p><p> </p>

---

 <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/163/discussion/#comment-2404784717'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by atravitz** [comment]
<p>done</p>

---

 <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/163/discussion/#comment-2404784750'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by atravitz** [comment]
<p>It was broken before this PR, but it's fixed in this version. I think it's a reviewnb bug that the entire link isn't highlighted here.</p>

---

 <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/163/discussion/#comment-2404784772'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by atravitz** [comment]
<p>I agree that it should be capitalized when in reference to the class type. I thought this was using "protocol" in the general sense. I think we can actually just rephrase like this:</p><p> </p><blockquote>As this involves both a relative binding free energy in solvent and complex phases, <code>RelativeHybridTopologyProtocol</code> will be used to build two separate <code>ProtocolDAG</code> (directed-acyclic-graph) classes, one for each phase.</blockquote>

---

 <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/163/discussion/#comment-2404784788'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by atravitz** [comment]
@IAlibay comments addressed, anything else before merging?

**Review by IAlibay — APPROVED** [review]
lgtm thanks!

**Pull request merged**